helm/observability-operator/templates/webhook/validatingwebhookconfiguration.yaml

```diff
@@ -0,0 +1,36 @@
+
+apiVersion: admissionregistration.k8s.io/v1
+kind: ValidatingWebhookConfiguration
+metadata:
+  labels:
+    {{- include "labels.common" . | nindent 4 }}
+  name: {{ include "resource.default.name" . }}-validating-webhook
+webhooks:
+- admissionReviewVersions:
+  - v1
+  clientConfig:
+    service:
+      name: {{ include "resource.default.name" . }}-webhook-service
+      namespace: {{ .Release.Namespace }}
+      path: /validate--v1-secret
+      port: 443
+  failurePolicy: Fail
```

Comment:
This means that no alertmanager config secret will be able to be created/updated if the webhook is down. Just making sure we are ok with this behavior.

Reply:
Do we want to allow something that will break alertmanager?